github/ci/prometheus/templates/grafana.yaml

```diff
@@ -26,6 +26,9 @@ spec:
         zone: ci
       containers:
         - name: grafana
+          securityContext:
+            runAsUser: 0
+            privileged: true
```

Comment:
I'm not sure that I like this... maybe we should allow some host paths in our cluster for unprivileged users.
@dhiller @rmohr what do you think?

Reply:
Same here. Didn't we talk about adding a PVC here BTW?

Reply:
@dhiller @danielBelenky Hello, if I use a PVC, what storage class can/should I use for it?